lang/test/org/partiql/lang/eval/builtins/functions/MathFunctionsTest.kt

```diff
@@ -0,0 +1,35 @@
+package org.partiql.lang.eval.builtins.functions
+
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ArgumentsSource
+import org.partiql.lang.eval.EvaluatorTestBase
+import org.partiql.lang.eval.builtins.ExprFunctionTestCase
+import org.partiql.lang.util.ArgumentsProviderBase
+
+class MathFunctionsTest : EvaluatorTestBase() {
+
+    @ParameterizedTest
+    @ArgumentsSource(MathFunctionsPassCases::class)
+    fun runPassTests(tc: ExprFunctionTestCase) = runEvaluatorTestCase(
+        tc.source,
+        expectedResult = tc.expectedLegacyModeResult
+    )
+
+    class MathFunctionsPassCases : ArgumentsProviderBase() {
+
+        override fun getParameters(): List<Any> = listOf(
```

Comment:
Nice set of tests! Can also add some tests
- using the Ion literal constructor (i.e. using backticks). For example: ``ceil(`1`)``  returns `1`
- testing with floats. For example: `ceil(1.1e0)` returns `2`
  - similarly, we may also need to define the semantics for special float values (i.e. `+inf`, `-inf`, and `NaN`)

Reply:
CEIL / FLOOR of +inf / -inf / NaN should be +inf / -inf / NaN respectively

Reply:
I will need to address this later after discussion

Reply:
I think some of the float special value tests still need to be added